src/Package/MSTest.Sdk/Sdk/Runner/Common.targets

```diff
@@ -26,4 +26,9 @@
     <MicrosoftTestingExtensionsCodeCoverageVersion Condition=" '$(MicrosoftTestingExtensionsCodeCoverageVersion)' == '' " >$(MicrosoftTestingExtensionsCodeCoverageVersion)</MicrosoftTestingExtensionsCodeCoverageVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <EnablePlaywright Condition=" '$(EnablePlaywright)' == '' ">false</EnablePlaywright>
```

Comment:
Shall this be in the sdk.props.template file?

Reply:
I would put both in the prop as defaults yep